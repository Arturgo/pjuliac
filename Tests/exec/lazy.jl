
function bad()
    println(div(1, 0))
    false
end
function good()
    println("good")
    false
end

println(false && bad())
println(true && good())
println(good() && bad())


