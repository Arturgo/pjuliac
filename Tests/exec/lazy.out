false
good
false
good
false
